cmd/envtool/user.json

```diff
@@ -0,0 +1,145 @@
+{
+  "$s": {
+    "p": {
+      "_id": {
+        "t": "string"
+      },
+      "credentials": {
+        "t": "object",
+        "$s": {
+          "p": {
+            "SCRAM-SHA-1": {
+              "t": "object",
+              "$s": {
+                "p": {
+                  "storedKey": {
+                    "t": "string"
+                  },
+                  "iterationCount": {
+                    "t": "int"
+                  },
+                  "salt": {
+                    "t": "string"
+                  },
+                  "serverKey": {
+                    "t": "string"
+                  }
+                },
+                "$k": [
+                  "storedKey",
+                  "iterationCount",
+                  "salt",
+                  "serverKey"
+                ]
+              }
+            },
+            "SCRAM-SHA-256": {
+              "t": "object",
+              "$s": {
+                "p": {
+                  "storedKey": {
+                    "t": "string"
+                  },
+                  "iterationCount": {
+                    "t": "int"
+                  },
+                  "salt": {
+                    "t": "string"
+                  },
+                  "serverKey": {
+                    "t": "string"
+                  }
+                },
+                "$k": [
+                  "storedKey",
+                  "iterationCount",
+                  "salt",
+                  "serverKey"
+                ]
+              }
+            },
+            "PLAIN": {
+              "t": "object",
+              "$s": {
+                "p": {
+                  "algo": {
+                    "t": "string"
+                  },
+                  "iterationCount": {
+                    "t": "long"
+                  },
+                  "hash": {
+                    "t": "binData",
+                    "s": 0
+                  },
+                  "salt": {
+                    "t": "binData",
+                    "s": 0
+                  }
+                },
+                "$k": [
+                  "algo",
+                  "iterationCount",
+                  "hash",
+                  "salt"
+                ]
+              }
+            }
+          },
+          "$k": [
+            "SCRAM-SHA-1",
+            "SCRAM-SHA-256",
+            "PLAIN"
+          ]
+        }
+      },
+      "user": {
+        "t": "string"
+      },
+      "db": {
+        "t": "string"
+      },
+      "roles": {
+        "t": "array",
+        "i": []
+      },
+      "userId": {
+        "t": "binData",
+        "s": 4
+      }
+    },
+    "$k": [
+      "_id",
+      "credentials",
+      "user",
+      "db",
+      "roles",
+      "userId"
+    ]
+  },
+  "_id": "admin.username",
+  "credentials": {
```

Comment:
Have we decided to use a default username/password?
Does MongoDB do the same? I'm asking because this makes me uncomfortable as it's one of the fragilities most attacked everywhere regarding security.

Reply:
This user needs to be the same as the user used in mongodb uri for our test and dev setup to work, for example https://github.com/FerretDB/FerretDB/blob/main/Taskfile.yml#L219]

This env-setup is for our development environment purpose so I'm not concerned with security, we have the same credentials visible in many other places :thinking:

Reply:
Or perhaps you mean we should not show hashed value?

Reply:
Got it! Would it make sense to have this named something else like testuser.json? Because from the path alone and no other context I can't know that it's only used for dev / tests. Physically not, though!

Reply:
No. I was wondering about the default value. It doesn't matter if it's hashed or not for me.